src/test/java/org/web3j/console/KeyImporterTest.java

```diff
@@ -30,26 +29,27 @@
     IODevice console = mock(IODevice.class);
 
     @Test
-    public void testSpecifyPrivateKey(@TempDir Path tempDirPath) {
-        prepareWalletCreation(SampleKeys.PRIVATE_KEY_STRING, tempDirPath);
+    public void testSpecifyPrivateKey() {
```

Comment:
So the core of the issue using this @TempDir annotation?

Reply:
Mac OS security model likely core issue